eng/pipelines/branch-cleanup.yml

```diff
@@ -38,17 +38,19 @@ jobs:
               -RepoId ${{ repo }}
               -BranchPrefix "sync-eng/common-" 
               -AuthToken $(azuresdk-github-pat)
-      - ${{ each repo in parameters.DailyBranchRepos }}:
-        - task: PowerShell@2
-          displayName: Clean Up Docs Daily Branches
-          condition: succeededOrFailed()
-          inputs:
-            pwsh: true
-            workingDirectory: $(System.DefaultWorkingDirectory)
-            filePath: $(System.DefaultWorkingDirectory)/eng/common/scripts/Delete-RemoteBranches.ps1
-            arguments: >
-              -RepoId ${{ repo }}
-              -BranchPrefix "daily"
-              -LastCommitOlderThan ((Get-Date).AddDays(-7))
-              -AuthToken $(azuresdk-github-pat)
+              -CentralRepoId "Azure/azure-sdk-tools"
+              -CentralPRRegex "^.*[^-]*-(?<PrNumber>\d+).*$"
```

Comment:
The centralRepoId is `Azure/azure-rest-api-specs-pr` in deleting branches creaed by sdk automation pipeline. Also, the regex doesn't match the branch name created by sdk autmation pipeline.
I think we need to add a new job here.